shared/src/models/map-image.ts

```diff
@@ -15,12 +15,16 @@ export class MapImage {
     @Type(() => ImageProperties)
     public readonly image: ImageProperties;
 
+    @IsBoolean()
+    public readonly locked: boolean;
```

Comment:
mountain

I think you must write a migration for this as previous states didn't have this property, and it isn't optional.

Reply:
There is a migration now ([8f2ce60](https://github.com/hpi-sam/digital-fuesim-manv/pull/545/commits/8f2ce60b9dc2e73956792294684b65a352cbbf4e))